### PR TITLE
Nano aod addition

### PR DESCRIPTION
**Pull request by LinaresToine**
Implement nanoAOD

### CODE REVIEW

**Comment by LinaresToine** [comment]
test syntax please

**Comment by germanfgv** [comment]
test syntax please

File: etc/ReplayOfflineConfiguration.py
Comment (germanfgv) [inline etc/ReplayOfflineConfiguration.py:1]: Please remove these changes from this PR. The proper replay and prod configs where updated in #4830

**Review by germanfgv — COMMENTED** [review]
Simple reset the Replay config so we can merge this into master. Thank you for your development

**Pull request merged**